custom_components/homewhiz/config_flow.py

```diff
@@ -96,6 +96,9 @@ async def async_step_select_bluetooth_device(
             if discovery_info.name.startswith("HwZ"):
                 self._discovered_bt_devices[address] = discovery_info.name
 
+        if len(self._discovered_bt_devices) == 0:
```

Comment:
If ``self._discovered_bt_devices`` is a list, `if self._discovered_bt_devices:` should be sufficient.

Reply:
I like to be explicit in my code